dist/index.html

```diff
@@ -2,19 +2,90 @@
 <html>
 <head>
   <meta charset="utf-8">
-  <title>My JavaScript App</title>
+  <title>TrekTrek</title>
+  <link rel="stylesheet" type="text/css" href="../src/css/foundation.css">
+  <link rel="stylesheet" type="text/css" href="../src/css/style.css">
   <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="shortcut icon" href="favicon.png">
 </head>
 <body>
   <header>
-
+    <h1>BackTREK<span class='float-right button' id='add-trip-button'>Add a trip</span></h1>
   </header>
-  <main>
+
+  <main class='row'>
+
+    <section id="status-messages">
+        <button class="clear float-right">
+          <img src="http://www.pvhc.net/img223/xeiwtpbbgpmyhtcnjvwt.png" alt="Clear">
+        </button>
+        <ul></ul>
+      </section>
+
+    <section class="columns large-6 small-12">
+      <button id='all-trips' class='button'>Trips</button>
+      <table>
+        <thead>
+          <th class="sort name">Name</th>
+          <th class="sort continent">Continent</th>
+          <th class="sort weeks">Weeks</th>
+          <th class="sort cost">Cost</th>
+          <th class="sort category">Category</th>
+        </thead>
+        <tbody id="trip-list">
+        </tbody>
+      </table>
+    </section>
+
+    <section class="columns large-6 small-12" id="trip-details"></section>
+
+    <div id='add-trip'>
+      <form class='add-trip-form'>
+        <h4>Add a trip</h4>
+        <input type='text' name='name' placeholder='Name' />
+        <input type='text' name='continent' placeholder='Continent' />
```

Comment:
A `select` element would make a better choice, not giving the user an option for an invalid continent.